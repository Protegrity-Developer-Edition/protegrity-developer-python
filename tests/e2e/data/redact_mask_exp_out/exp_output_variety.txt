On [DATE_TIME], a financial transaction of $1,250.75 was initiated from account number [ACCOUNT_NUMBER] using routing number 021000021. The payment was made by [PERSON], aged [DATE_TIME], residing at [STREET|LOCATION|BUILDING], [LOCATION], and currently working at Acme Corp in [CITY], [LOCATION]. His email address is [EMAIL_ADDRESS] and his contact number is [PHONE_NUMBER]. The transaction was verified using credit card number [CREDIT_CARD] with CVV [CREDIT_CARD_CVV] and was linked to the Ethereum wallet address [BITCOIN_ADDRESS] and [NRP] address [CRYPTO].
Additional identifiers include [PERSON] number 1234 5678 9012, PAN [IN_PAN], passport number [ID_CARD], and driver’s license [DRIVER_LICENSE]. The user’s IP address was [IP_ADDRESS] and MAC address [MAC]. The transaction occurred at [TIME] and was logged under username [USERNAME]. The user’s date of birth is [DATE_OF_BIRTH] and their gender is [GENDER]. The location coordinates were 3[GEO_CCORDINATE] and the IBAN used was [IBAN_CODE] with BIC DEUTDEFF.
Other identifiers include [NRP] ABN [AU_ABN], ACN [DATE_TIME], Medicare number [SOCIAL_SECURITY_NUMBER], and TFN [DATE_TIME]. The user also provided their [NRP] fiscal code [DRIVER_LICENSE], [NRP] [DRIVER_LICENSE|PERSON], and [LOCATION] NRIC [ID_CARD]. The organization OpenAI Research was listed as the employer, and the user’s password was [PASSWORD]. The user’s address also included [SECONDARYADDRESS], [STREET] [STREET], [COUNTY], [STATE], ZIP code [ZIP_CODE]. The transaction was recorded on the URL [USERNAME|URL] and timestamped at [DATE_TIME].