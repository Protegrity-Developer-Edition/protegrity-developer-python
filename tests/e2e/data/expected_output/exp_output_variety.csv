ABA_ROUTING_NUMBER,ACCOUNT_NAME,ACCOUNT_NUMBER,AGE,AMOUNT,AU_ABN,AU_ACN,AU_MEDICARE,AU_TFN,BIC,BITCOIN_ADDRESS,BUILDING,CITY,COMPANY_NAME,COUNTRY,[LOCATION],CREDIT_CARD,CREDIT_CARD_CVV,CRYPTO,CURRENCY,CURRENCY_CODE,CURRENCY_NAME,CURRENCY_SYMBOL,DATE,[PERSON],DATE_TIME,DRIVER_LICENSE,EMAIL_ADDRESS,ES_NIE,ES_NIF,ETHEREUM_ADDRESS,FI_PERSONAL_IDENTITY_CODE,GENDER,GEO_CCORDINATE,IBAN_CODE,ID_CARD,IN_AADHAAR,IN_PAN,IN_PASSPORT,IN_VEHICLE_REGISTRATION,IN_VOTER,IP_ADDRESS,IPV4,IPV6,IT_DRIVER_LICENSE,IT_FISCAL_CODE,IT_IDENTITY_CARD,IT_PASSPORT,LITECOIN_ADDRESS,LOCATION,MAC,MEDICAL_LICENSE,NRP,ORGANIZATION,PASSPORT,PASSWORD,PERSON,PHONE_NUMBER,PIN,PL_PESEL,SECONDARY_ADDRESS,[LOCATION],[LOCATION],STATE,STREET,TIME,TITLE,UK_NHS,URL,[LOCATION],US_DRIVER_LICENSE,US_ITIN,US_PASSPORT,US_SSN,USERNAME,ZIP_CODE
028354966,[PERSON] Gordon,285856949268,36,55.72,43333959366,598339491,0916103982,372118829,BXDSGB8AWBF,[CRYPTO],37771,Gouldborough,Leon-Davis,[LOCATION],Hawaii,[CREDIT_CARD],903,0x17377b283f6fedaa123ec54094c4eadceb331c1c,[NRP] dollar,KZT,[LOCATION] dollar,[TITLE]1[DATE_OF_BIRTH]1[DATE_OF_BIRTH]2017-05-27T10:06:23.035470,047916207,n[EMAIL_ADDRESS]X[ID_CARD]056179129,0xd02874ede68ef9a877af3a2cad02f3fe0b11f47c,66929705225,M[GENDER]"-30.9338805,135.477391",G[IBAN_CODE]535090610,266245112781,8321728551,299272668,M[IN_VEHICLE_REGISTRATION]XYZ1234567,2[IP_ADDRESS]9[IP_ADDRESS],5d[IP_ADDRESS]7973178677,R[DRIVER_LICENSE]3[DRIVER_LICENSE]E[DRIVER_LICENSE]LcHKjUoU7zjvT9Z6ZzvZzvZzvZzvZzvZzv,"776 W[PERSON]
[LOCATION], [STATE] [ZIP_CODE]",30:ec:25:7f:f9:ff,MED123456,7595915299,Burch-Thomas,[AU_ACN],[PASSWORD],[PHONE_NUMBER|PERSON]9,8829,83639969954,Suite 195,[ID_CARD],[ID_CARD],[AU_ACN],[STATE],0491 [PERSON|STREET]. [TIME|DATE_TIME]ITLE],[DRIVER_LICENSE],http://www.powell.info/,[ID_CARD],125819798,299824289,[DRIVER_LICENSE],783907591,[USERNAME],24706
[AU_ACN],Dustin [ID_CARD|PERSON]REDIT_CARD],691,0x4744fe7108e6d3be9b526c2551f2dd9f7c84e1d5,Bermudian dollar,[CURRENCY_CODE],[PERSON|LOCATION],LD,1994-03-16,1916-06-11,2021-09-23T23:32:40.994143,[AU_TFN],[EMAIL_ADDRESS],[ID_CARD],750065446,0x4be08173e519bb095e8a5e321c4b29f58cababbe,61993811556,[GENDER],"64.3792625,137.918475",[IBAN_CODE],[AU_TFN],981643951877,[DRIVER_LICENSE|UK_NHS],[DRIVER_LICENSE],[IN_VEHICLE_REGISTRATION],[PASSPORT],[IP_ADDRESS],[IP_ADDRESS],[IP_ADDRESS],[DRIVER_LICENSE],[DRIVER_LICENSE],[DRIVER_LICENSE],[PASSPORT],LcHKjUoU7zjvT9Z6ZzvZzvZzvZzvZzvZzv,"USNS [LOCATION]
FPO AE 44904",14:94:09:5b:87:81,MED123456,0116473795,[PERSON],[DRIVER_LICENSE],[PASSWORD],[PHONE_NUMBER|PERSON] 933,[ID_CARD],[ID_CARD],[PASSPORT],[STATE],2501 [PERSON|STREET] 932,[TIME],[TITLE],[UK_NHS],http://roberts.biz/,788622632701,[AU_TFN],[AU_ACN],700363887,[AU_TFN],blake58,94043
904341312,Samantha Goodwin,[ID_CARD]6,75,5406.83,[ID_CARD],962595184,9[ID_CARD],366292671,[DRIVER_LICENSE],[CRYPTO],[BUILDING],West Joseph,"Simmons, [LOCATION] and [LOCATION],Maine,4717548774369840933,752,0x6f90aeb2a15281531c20473268966c740a082a9c,[NRP] rand,[CURRENCY_CODE],[NRP] convertible peso,E,2019-09-30,1986-01-03,1991-11-14T00:08:56.206220,573101568,[EMAIL_ADDRESS],[ID_CARD],003232402,0xd6f1c0fedd8a1759d114a75b10c5a35de02f22e3,58762193545,[GENDER],"28.628268,174.288786",[IBAN_CODE],216810677,338346897508,[DRIVER_LICENSE],[DRIVER_LICENSE],[IN_VEHICLE_REGISTRATION],XYZ1234567,[IP_ADDRESS],[IP_ADDRESS],[IP_ADDRESS]6,5224989564,[DRIVER_LICENSE],[AU_ACN],[DRIVER_LICENSE],LcHKjUoU7zjvT9Z6ZzvZzvZzvZzvZzvZzv,"07215 [PERSON] Way
[PERSON], [STATE] [ZIP_CODE]",74:f2:f3:29:37:f2,MED123456,4646878647,"Rogers, [LOCATION] and Rodriguez",194195082,[PASSWORD],Martha Harris,[PHONE_NUMBER],[ZIP_CODE],[ID_CARD],Suite 849,[ID_CARD],[ID_CARD],076286386,Louisiana,[BUILDING] [PERSON|STREET],[TIME],[TITLE],1454220782,http://www.perry.com/,[ID_CARD],[AU_TFN],017441426,216970579,150271356,[USERNAME],07222
415315934,Robert Cohen,[ID_CARD],71,567.52,80111667727,028476378,[ID_CARD],[PASSPORT],[DRIVER_LICENSE],[CRYPTO],[ZIP_CODE],East Rachaelport,Johnson Inc,[STATE],[STATE|LOCATION]TREET] dollar,$,[DATE_OF_BIRTH],[DATE_OF_BIRTH],2001-05-30T08:06:12.082557,732948494,[EMAIL_ADDRESS],[ID_CARD],539204482,0xd4a7acbc81c275370d29124a61bbcf816f3cf804,99460823804,[GENDER],"-40.693165,40.477252",[IBAN_CODE],187016318,352927130839,[DRIVER_LICENSE|UK_NHS],[DRIVER_LICENSE],[IN_VEHICLE_REGISTRATION],XYZ1234567,[IP_ADDRESS],[IP_ADDRESS]2,5a[IP_ADDRESS],[DRIVER_LICENSE],[DRIVER_LICENSE],437261711,771527624,LcHKjUoU7zjvT9Z6ZzvZzvZzvZzvZzvZzv,"Unit [PERSON] 8638
DPO AP 30002",28:55:62:8b:36:12,MED123456,1337392188,Bentley Ltd,651647204,[PASSWORD],[PERSON],[PHONE_NUMBER],[ZIP_CODE],[ID_CARD],Apt. 987,[ID_CARD],[ID_CARD],[PASSPORT],[PERSON],[PERSON|STREET] [TIME|DATE_TIME]TLE],2911400733,https://www.baker-pruitt.com/,[IN_AADHAAR],[AU_TFN],878399848,279805958,089895101,[USERNAME],78459
561919344,Edward Marquez,[ID_CARD],37,2022.53,[ID_CARD],[AU_ACN],5065427588,830010884,[DRIVER_LICENSE],[CRYPTO],[ZIP_CODE],[CITY],Schultz LLC,[LOCATION],[STATE],[CREDIT_CARD],012,0x0014566ea90778ffdf736b18ed7e3c2d82817c74,Salvadoran colón,SOS,[NRP] riyal,$,1993-08-09,1921-10-18,1978-04-01T00:32:04.136241,950332255,[EMAIL_ADDRESS],[ID_CARD],811775494,0x68f0c51fc65a3f40917aea7528c101f3af753f5b,86277130141,[GENDER],"-50.894331,102.612151",[IBAN_CODE],142881497,[IN_AADHAAR],3005548707,[AU_ACN],[IN_VEHICLE_REGISTRATION],XYZ1234567,[IP_ADDRESS],[IP_ADDRESS]5,5e[IP_ADDRESS],[UK_NHS],[DRIVER_LICENSE],[DRIVER_LICENSE],[DRIVER_LICENSE],LcHKjUoU7zjvT9Z6ZzvZzvZzvZzvZzvZzv,"69228 Jessica Well
[PERSON], [STATE] [ZIP_CODE]",18:ae:f7:50:31:f4,MED123456,2584671154,Holloway-Webb,[PASSWORD|DATE_TIME] Bowers,[PHONE_NUMBER],4367,92242845648,[STREET],[ID_CARD],[ID_CARD],[PASSPORT],Maryland,060 [PERSON|STREET] [TIME|PERSON]TLE],8404407255,http://www.hartman-alvarez.org/,470940484146,832248030,166265142,K62890681,939690780,[USERNAME],49692
252952570,Nicole Watson,[ID_CARD],60,8649.93,[ID_CARD],[AU_TFN],[ID_CARD],215796844,[DRIVER_LICENSE],[CRYPTO],9733,South Sandra,"Richardson, [PERSON] and [PERSON],[PERSON|LOCATION] dram,HNL,[NRP] cedi,[TITLE],2020-04-15,2022-05-09,1988-06-24T11:32:30.525397,[AU_TFN],[EMAIL_ADDRESS],[ID_CARD],500059729,0xc279c05ba863dec56b54feb98ab3d082f2285ad0,14224480331,[GENDER],"-55.131472,-36.042962",[IBAN_CODE],937252951,129868526307,[DRIVER_LICENSE]1,[DRIVER_LICENSE|AU_ACN],[IN_VEHICLE_REGISTRATION],XYZ1234567,[IP_ADDRESS],[IP_ADDRESS],[DRIVER_LICENSE|PERSON|IP_ADDRESS]
FPO AP 14591",ba:a2:45:4c:9e:33,MED123456,[ID_CARD],"Smith, [PERSON] and Jones",[DRIVER_LICENSE]7,6oR27FuX(7,Cole Brown PhD,[ID_CARD],9662,[ID_CARD],Suite 775,[ID_CARD],[ID_CARD],[PASSPORT],Oklahoma,0354 [STREET] 947,[TIME],[TITLE],[DRIVER_LICENSE],http://james.com/,[ID_CARD],309294426,[AU_ACN],R08653744,[DRIVER_LICENSE|AU_TFN],[USERNAME],14934
910282454,David [DRIVER_LICENSE|PERSON]ATION],Montana,[CREDIT_CARD],593,0x48584b429ab0f78a11de803706f600aee0d2b43f,[NRP] new shekel,[PERSON],[NRP] rufiyaa,$,1982-09-07,1993-10-22,1988-05-22T18:04:00.959271,657298306,[EMAIL_ADDRESS],[ID_CARD],099451039,0x87495b93057d426acaf04bf65025b00c390983e3,[DRIVER_LICENSE],[GENDER],"8.461279,157.241694",[IBAN_CODE],650469954,996147447047,[UK_NHS],[DRIVER_LICENSE],[IN_VEHICLE_REGISTRATION],XYZ1234567,[IP_ADDRESS],[IP_ADDRESS],[PERSON|IP_ADDRESS],[DRIVER_LICENSE],[DRIVER_LICENSE],[DRIVER_LICENSE],Q27906615,LcHKjUoU7zjvT9Z6ZzvZzvZzvZzvZzvZzv,"879 [STREET] Apt. 652
[LOCATION], [ZIP_CODE|LOCATION|STATE]:b9:b5:51:7b:73,MED123456,0412766991,Powell and Sons,[DRIVER_LICENSE],[PASSWORD],[PHONE_NUMBER|PERSON]36,[ID_CARD],[ID_CARD],[PASSPORT],[STATE],33211 [LOCATION|STREET] 692,[TIME],Ind.,[DRIVER_LICENSE],https://gillespie.com/,439185959170,370670973,075810717,P93048684,7[DRIVER_LICENSE],[USERNAME],84154
[AU_ACN|DATE_TIME]PERSON|LOCATION],[STATE],[CREDIT_CARD],711,0xd4352f7622d9be536859e0da660ad3831c15ecfe,[NRP] dalasi,SHP,[NRP] quetzal,؋,1975-04-09,1995-07-01,2011-02-26T20:13:57.593266,005134296,[EMAIL_ADDRESS],[ID_CARD],[AU_TFN],0xbc4594752792dd0c23954d811576a71ad37bee9f,[AU_ABN],[GENDER],"-28.214933,-157.286518",[IBAN_CODE],761503501,297941476334,[DRIVER_LICENSE],[DRIVER_LICENSE],[IN_VEHICLE_REGISTRATION],XYZ1234567,[IP_ADDRESS],[IP_ADDRESS],[IP_ADDRESS],[UK_NHS],[DRIVER_LICENSE],[DRIVER_LICENSE],289183639,LcHKjUoU7zjvT9Z6ZzvZzvZzvZzvZzvZzv,"7321 [PERSON]
New Matthewmouth, [STATE] [ZIP_CODE]",68:bb:02:f4:e4:29,MED123456,[ID_CARD],Willis Inc,[DRIVER_LICENSE],[PASSWORD],[PHONE_NUMBER|PERSON],[ID_CARD],Apt. 827,[ID_CARD],[ID_CARD],[AU_TFN],[STATE],0284 [PERSON] Bypass,[TIME],[TITLE],6749355735,http://hill.com/,[ID_CARD],527680530,774848469,[AU_TFN],009868988,[USERNAME],87015